Packs/PaloAltoNetworks_IoT3rdParty/Scripts/SendPANWIoTDevicesToCiscoISE/README.md

```diff
@@ -0,0 +1,38 @@
+This script takes in PANW IoT cloud devices as input and exports them as Endpoints with custom attributes in Cisco ISE
```

Comment:
This script takes PANW IoT cloud devices as input and exports them as endpoints with custom attributes in Cisco ISE.